fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/digitalcash/DigitalCashActivity.java

```diff
@@ -233,59 +101,83 @@ public static DigitalCashViewModel obtainViewModel(FragmentActivity activity) {
   }
 
   public void setupNavigationBar() {
-    bottomNavigationView = findViewById(R.id.digital_cash_nav_bar);
-    bottomNavigationView.setOnItemSelectedListener(
+    navbar = findViewById(R.id.digital_cash_nav_bar);
+    navbar.setOnItemSelectedListener(
         item -> {
```

Comment:
would it be helpful to define the content of the listener in its own method ?

Reply:
I would argue the only thing the method does is getting the navbar and adding a listener to it. IMO it is therefore fine to define said listener there directly.
Then again if you have strong feelings about it I have no problem changing it